src/Microsoft.Diagnostics.Monitoring.RestServer/Microsoft.Diagnostics.Monitoring.RestServer.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Target netcoreapp3.1 and ns2.0, that'll make it easy to set the flag that turns off endpoint routing (or change how we wire up the application in general).

Reply:
I think the only downside of that is that we have to ship two tools right, one for 2.1 and one for 3.1.

Reply:
Why? It's a single package that multi-targets right? Pretty sure we have other tools that do this today (or used to).

Reply:
If i build dotnet-monitor against netcoreapp 2.1 and reference restserver netstandard 2.0, i still can't run that against a 3.1 runtime correct? i would need to build a separate netcoreapp 3.1 dotnet-monitor.

Reply:
There's a way to do this but lets stick with the reflection for now and address this separately.